### PR TITLE
scipt-file

### PR DESCRIPTION
**Pull request by wazshaiwayana**
hey guys, I created a script file and got the pixabay api working. let me know if you want anything changed .

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -20,8 +22,8 @@ <h1>Travoogle</h1>
   <div class = "row " id="searchDiv">
       <div class="col-md-4 box"> 
         <div class="card">  
-            <input type="text" id="search"class="form-control" placeholder="Travel to.....">
-            <button id="submit" class="btn btn-warning">Search</button>
```
Comment (acwilliamson97) [inline index.html]: we still need this button with the id of "search" for my js file to work.

Comment (acwilliamson97) [reply index.html:24]: i would recommend adding a class to the original button rather than deleting this button.

**Comment by wazshaiwayana** [comment]
can you switch it to this id instead? i worked on this with a tutor and
they said that "search" is already a built in class so he said to change
the class name .

**Comment by wazshaiwayana** [comment]
im sorry i meant submit. search should still be there.

**Review by acwilliamson97 — CHANGES_REQUESTED** [review]
This HTML file does not reflect the changes I added earlier. I cannot comfortably approve this merge request. The script tags belong at the bottom of the html also.  From here on out, any changes made to index.html need to ensure that previous pushes are still working (i.e. appending the correct info to the correct tags).

**Review by acwilliamson97 — APPROVED** [review]

**Pull request merged**